### PR TITLE
feat(*): Optimism to OP Mainnet, batch 2

### PR DESCRIPTION
**Pull request by qbzzt**

### CODE REVIEW

File: src/docs/developers/bedrock/explainer.md [hunk 1]
```diff
@@ -8,11 +8,11 @@ meta:
 
 ![Bedrock](https://dev.optimism.io/content/images/size/w2000/2022/12/bedrock-BLUE.jpg)
 
-Bedrock is the name of the first ever official release of the OP Stack, which is a set of free and open-source modular components that work together to power Optimism.
+Bedrock is the name of the first ever official release of the OP Stack, which is a set of free and open-source modular components that work together to power Optimism blockchains such as OP Mainnet.
```
Comment (OPMattie) [inline src/docs/developers/bedrock/explainer.md:11]: Remove 'Optimism' from 'Optimism blockchains such as OP Mainnet'. Should just be 'blockchains such as OP Mainnet".

File: src/docs/developers/bedrock/wallet-upgrade.md [hunk 2]
```diff
@@ -44,8 +44,8 @@ To enable your users to select a priority fee, you can [build a priority fee est
 (https://docs.alchemy.com/docs/how-to-build-a-gas-fee-estimator-using-eip-1559).
 If you already have estimating code you use for L1 Ethereum, you can just use that.
 
-Note that on Optimism the priority fee tends to be very low. 
-As I am writing this, a priority fee of 0.001 gwei is sufficient ([see here](https://optimism.io/gas-tracker) to get the current values).
+Note that on OP Mainnet the priority fee tends to be very low. 
+As I am writing this, a priority fee of 0.2 gwei is sufficient ([see here](https://optimism.io/gas-tracker) to get the current values).
```
Comment (OPMattie) [inline src/docs/developers/bedrock/wallet-upgrade.md:48]: Is 0.2 gwei correct? This is higher than what was previously noted here

File: src/docs/useful-tools/explorers.md [hunk 3]
```diff
@@ -14,15 +14,15 @@ We have a Blockscout explorer for the [Goerli testnet](https://blockscout.com/op
 
 ## Etherscan
 
-We have Etherscan explorers for the [Optimism mainnet](https://explorer.optimism.io) and the  [Optimism  Goerli testnet](https://goerli-explorer.optimism.io).
+We have Etherscan explorers for the [OP Mainnet](https://explorer.optimism.io) and the  [OP Goerli](https://goerli-explorer.optimism.io).
```
Comment (OPMattie) [inline src/docs/useful-tools/explorers.md:17]: Minor - seems like there are two spaces before the OP Goerli link

File: src/docs/useful-tools/providers.md [hunk 4]
```diff
@@ -50,52 +50,52 @@ Its API takes only a few lines of code to get up and running. Alchemy isn't just
...
 
-- Optimism Mainnet
-- Optimism Goerli
+- OP Mainnet
+- OP Goerli
 
 ## BlockSpaces
 
-[BlockSpaces](https://www.blockspaces.com/web3-infrastructure) is a multiweb integration platform that is currently offering Web3 infrastructure for free, up to 10 million transactions supported DAILY. To get started, click [here](https://www.blockspaces.com/web3-infrastructure) and leave an email address. If being able to integrate your Optimism dApp to other web2 business platforms (like Quickbooks) interestes you, make sure to sign up for our Multiweb integration waitlist.
+[BlockSpaces](https://www.blockspaces.com/web3-infrastructure) is a multiweb integration platform that is currently offering Web3 infrastructure for free, up to 10 million transactions supported DAILY. To get started, click [here](https://www.blockspaces.com/web3-infrastructure) and leave an email address. If being able to integrate your OP dApp to other web2 business platforms (like Quickbooks) interestes you, make sure to sign up for our Multiweb integration waitlist.
```
Comment (OPMattie) [inline src/docs/useful-tools/providers.md:75]: OP dApp should be OP Mainnet dapp

**Review by OPMattie — APPROVED** [review]

File: src/docs/developers/bedrock/wallet-upgrade.md (hunk 2 above)
Comment (qbzzt) [reply src/docs/developers/bedrock/wallet-upgrade.md]: It was right when I wrote it. Now it's 0.0002 gwei, which I agree is a much better number to have in the docs.

**Pull request merged**